src/main/java/io/tackle/controls/resources/StakeholderAddResource.java

```diff
@@ -0,0 +1,42 @@
+package io.tackle.controls.resources;
+
+import io.tackle.controls.entities.Stakeholder;
+import org.hibernate.exception.ConstraintViolationException;
+import org.jboss.resteasy.links.LinkResource;
+
+import javax.persistence.PersistenceException;
+import javax.transaction.Transactional;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+
+import static javax.ws.rs.core.Response.Status.*;
+
+@Path("stakeholder")
+public class StakeholderAddResource {
```

Comment:
How come we had to add this class here?

Reply:
Without this change, the tests inside this folder will fail https://github.com/konveyor/tackle-controls/tree/main/src/test/java/io/tackle/controls/resources/issues
 What this is basically doing is to have a custom method for the POST operation and disabling the out of the box implementation with this line of code https://github.com/konveyor/tackle-controls/pull/195/files#diff-2a67e8f9d78f7c1631bc0afeea9d91046dad9a4b0dd8cbe004b7acbff780ce56R17


The main problem is that the "Out of the box" implementation of resources coming from `io.quarkus.hibernate.orm.rest.data.panache.PanacheEntityResource` here https://github.com/konveyor/tackle-controls/blob/main/src/main/java/io/tackle/controls/resources/StakeholderResource.java#L13 doesn't work with Entities that present a mix of ManyToMany ManyToOne and OneToMany associations.

There is a precedent of a similar issue where we needed to remove the out of the box implementation an write our own code here https://github.com/konveyor/tackle-controls/blob/main/src/main/java/io/tackle/controls/resources/StakeholderUpdateResource.java#L95